README.md

```diff
@@ -37,3 +37,31 @@ The relayer consists of the following components:
 <div align="center">
   <img src="resources/relayer-diagram.png?raw=true">
 </div>
+
+## Testing
+
+---
+
+### Unit tests
+
+Unit tests can be ran locally by running the command in root of the project:
+
+```bash
+./scripts/test.sh
+```
+
+### E2E tests
+
+E2E tests are ran as part of CI, but can also be ran locally. To run the E2E tests locally, you need to have the `avalanchego` build path and vm binary set up. For example with [subnet-evm](https://github.com/ava-labs/subnet-evm):
+
+```bash
+cd subnet-evm
+BASEDIR=/tmp/e2e-test AVALANCHEGO_BUILD_PATH=/tmp/e2e-test/avalanchego ./scripts/install_avalanchego_release.sh
+./scripts/build.sh /tmp/e2e-test/avalanchego/plugins/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
```

Comment:
To get the tests to work on my ubuntu box, I had to use a different directory other than `/tmp` to install avalanchego. I don't think my system will let me run executables from there. I just used '~/tmp` instead for all these steps.